README.md

```diff
@@ -132,12 +116,9 @@ The easiest way to develop in docker while still using an IDE is to use VSCode a
     1. [VSCode](https://code.visualstudio.com/)
```

Comment:
In the line above a couple nit-picky things.

The easiest way to develop in docker while still using an IDE is to use VSCode. Follow the steps below to develop on this repo in a docker container.